### PR TITLE
Fix writestory.php

### PR DESCRIPTION
**Pull request by iffy**
This is a pull request.  It's a request for you to pull some changes I've made (in a branch called `fix-writestory.php`) into your `master` branch.  If you decide to merge the pull request, it will update your `master` branch on GitHub.  Then you can `git fetch` the changes from GitHub and finally `git merge origin/master` to update your local copy.

You can see what the changes are by clicking on "Files changed" tab above.

### CODE REVIEW

File: public/guest.php [hunk 1]
```diff
@@ -24,8 +24,10 @@ public static function find_all() {
   
 public static function find_by_name($name) {
 		global $database;
-		$sql = ("select * from " .self::$table_name. " where name = {$name} ");		
-		return $result_set = $database->query($sql);
+		$sanitized_name = $database->escape_value($name);
+		$sql = ("select * from " .self::$table_name. " where username = '{$sanitized_name}' ");
```
Comment (iffy) [inline public/guest.php]: I essentially just change `name` to `username` and put single quotes around the sanitized name.

Comment (jebarlow) [inline public/guest.php]: I would never have seen this, thank you. Now on to the next issue - I hope
you are still having fun.

On Sat, Jan 10, 2015 at 8:49 PM, Matt Haggard <notifications@github.com>
wrote:

> In public/guest.php
> <https://github.com/jebarlow/sincerelyyours/pull/1#discussion-diff-22764107>
> :
>
> > @@ -24,8 +24,10 @@ public static function find_all() {
> >
> >  public static function find_by_name($name) {
> >  		global $database;
> > -		$sql = ("select * from " .self::$table_name. " where name = {$name} ");		
> > -		return $result_set = $database->query($sql);
> > +		$sanitized_name = $database->escape_value($name);
> > +		$sql = ("select * from " .self::$table_name. " where username = '{$sanitized_name}' ");		
>
> I essentially just change name to username and put single quotes around
> the sanitized name.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/jebarlow/sincerelyyours/pull/1/files#r22764107>.
>



-- 
Eddie

**Comment by iffy** [comment]
To merge this, click the "Merge pull request" button.

**Comment by jebarlow** [comment]
I marked them as closed and selected the Pull Request Button but it did not
look like anything happened, I am assuming this is on GitHub.


On Sat, Jan 10, 2015 at 9:17 PM, Matt Haggard <notifications@github.com>
wrote:

> To merge this, click the "Merge pull request" button.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/jebarlow/sincerelyyours/pull/1#issuecomment-69483108>.
>



-- 
Eddie

**Comment by iffy** [comment]
Closing means "I don't want this change" but there is a green button labeled "Merge pull request" if the pull request isn't closed.  But I think you got the changes into master, so it's good.

**Pull request closed without merging**